diesel/src/macros/insertable.rs

```diff
@@ -199,6 +199,24 @@ macro_rules! _Insertable {
                 ,)+)
             }
         }
+
+    } __diesel_parse_as_item! {
+        impl<$($lifetime,)* 'insert, Op, Ret> $crate::query_builder::insert_statement::IntoInsertStatement<$table_name::table, Op, Ret>
+            for &'insert $struct_ty where
+                $('insert: $lifetime,)*
+                $crate::query_builder::insert_statement::InsertStatement<$table_name::table, &'insert $struct_ty, Op, Ret>: $crate::query_builder::AsQuery,
```

Comment:
I have observed that the compiler often doesn't care if I do `$lifetime: 'insert` or `'insert: $lifetime` though.